packages/@ama-sdk/schematics/schematics/typescript/core/index.ts

```diff
@@ -107,19 +108,19 @@ export function ngGenerateTypescriptSDK(options: NgGenerateTypescriptSDKCoreSche
    */
   const updateSpec = (tree: Tree, _context: SchematicContext) => {
     const specContent = readFileSync(specPath).toString();
-    if (tree.exists('/readme.md')) {
+    if (tree.exists(path.join(targetPath, 'readme.md'))) {
```

Comment:
`path.join(targetPath, 'readme.md')` could be extracted in a constant, as it is used in multiple places
same thing with `path.join(targetPath, 'swagger-spec.yaml')`